types/stripe/index.d.ts

```diff
@@ -508,29 +860,6 @@ declare namespace Stripe {
                 };
             };
 
-            /**
-             * The text that will appear on credit card statements by default if a charge is
-             * being made directly on the account.
-             */
-            statement_descriptor?: string;
-
-            /**
-             * A publicly shareable email address that can be reached for support for this
-             * account
-             */
-            support_email?: string;
```

Comment:
It should also be noted that these properties were moved rather than just deleted and therefore should be added back in their new place. But fine leaving that out of this PR

Reply:
`support_email` and other properties already exist in the `business_profile` object at the top of the `IAccountShared` interface [here](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/HEAD/types/stripe/index.d.ts#L283) in line with the API.

Reply:
Hrmm okay, but others such as `settings.branding.primary_color/logo/icon` etc. haven't been re-added

Reply:
They're all still under `settings.branding` of the `IAccountShared` object in [master](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/HEAD/types/stripe/index.d.ts#L376) and haven't moved in my PR. Looks like there was [another PR merged](https://github.com/DefinitelyTyped/DefinitelyTyped/commit/a408d42b1add1b32a21c9454eab6b529112e9d91) containing half of the `IAccountShared` properties and not clearing up the old ones.